package.json

```diff
@@ -41,7 +41,7 @@
     "archiver": "^5.1.0",
     "camelcase": "^6.2.0",
     "chalk": "4.1.0",
-    "chokidar": "^3.4.3",
+    "chokidar": "3.4.3",
```

Comment:
this was needed, because I got errors on macOS.
see: https://github.com/fsevents/fsevents/issues/355